src/raw/unix.rs

```diff
@@ -163,17 +177,26 @@ pub mod Type {
     pub const CLOEXEC: c_int = SOCK_CLOEXEC;
 }
 
-#[allow(non_snake_case)]
+#[allow(non_snake_case, non_upper_case_globals)]
 ///Socket protocol
 pub mod Protocol {
     use super::libc::*;
     pub const NONE: c_int = 0;
-    pub const ICMP: c_int = 1;
+    pub const ICMPv4: c_int = 1;
     pub const TCP: c_int = 6;
     pub const UDP: c_int = 17;
-    pub const ICMPV6: c_int = 58;
+    pub const ICMPv6: c_int = 58;
 }
 
+#[allow(non_snake_case)]
+///Possible flags for `accept()`
+pub mod AcceptFlags {

```

Comment:
What do you think about making it enum with bit flags?
This way you wouldn't need to check validity of flags in accept function.

E.g. https://docs.rs/hoedown/6.0.0/hoedown/struct.Extension.html